linux/debian/control

```diff
@@ -9,7 +9,7 @@ Build-Depends: debhelper (>= 13),
                gcc (>=4:8.0.0~),
                g++ (>=4:8.0.0~),
                golang (>=2:1.18~),
-               cargo (>=1.75),
+               cargo (>=1.82),
```

Comment:
This will cause a failure to build when submitted to the launchpad PPA build servers. The reason we set the minimum version to 1.75 here is because that's the version of `cargo` available from the Ubuntu security updates for Ubuntu/Jammy.

There should be workaround available somehow, and in theory we should be able to install an official 1.82 version (see: https://documentation.ubuntu.com/ubuntu-for-developers/reference/availability/rust/)

Sometimes a backport is available, in which case specifying the dependency as: `cargo (>= 1.82) | cargo-1.82` might do the trick. We have had to do similar workarounds in the past like this (eg: see #6855 for how `gcc-8` compatibility was patched into Ubuntu/Bionic).

Reply:
If this works, all the hacking to get `rustup` to work should become unnecessary, which would clean up this PR significantly.